packages/playwright/src/generators/accessibility/generator.spec.ts

```diff
@@ -25,30 +25,55 @@ jest.mock('@nrwl/devkit', () => {
                         'test',
                         {
                             root: '',
-                            sourceRoot: `${projectNameE2E}/src`,
+                            sourceRoot: `${projectName}/src`,
                             name: 'test',
                         },
                     ],
+                    [
+                        'test-e2e',
+                        {
+                            root: '',
+                            sourceRoot: `${projectNameE2E}/src`,
+                            name: 'test-e2e',
+                        },
+                    ],
                 ]),
         ),
     };
 });
 
 describe('playwright accessibility generator', () => {
     let appTree: Tree;
-    let options: AccessibilityGeneratorSchema;
 
     beforeEach(() => {
-        options = {
-            project: projectName,
-            accessibility: true,
-        };
         appTree = createTreeWithEmptyWorkspace();
 
-        addStacksAttributes(appTree, options.project);
+        addStacksAttributes(appTree, projectName);
+    });
+
+    it('should error if the project does not supported', async () => {
+        const options: AccessibilityGeneratorSchema = {
+            project: 'non-supported-project',
```

Comment:
We have not mocked this as a return value for getProjects(), so would error before getting to the code we want to check
```suggestion
            project: 'test',
```